src/wizard/dotnet/aspnet.md

```diff
@@ -0,0 +1,64 @@
+---
+name: ASP.NET
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/aspnet/
+support_level: production
+type: framework
+---
+
+Install the **NuGet** package:
+
+Package Manager:
+
+```shell
+Install-Package Sentry -Version {{ packages.version('sentry.dotnet') }}
+```
+
+<div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
+    <div class="alert-body content-flush-bottom">
+        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
+    </div>
+</div>
+
+Add Sentry to `Global.asax.cs`:
+
+```csharp
+using System.Web;
+using Sentry;
+
+public class MvcApplication : HttpApplication
+{
+    private IDisposable _sentry;
+
+    protected void Application_Start()
+    {
+        // Initialize Sentry to capture AppDomain unhandled exceptions and more.
+        _sentry = SentrySdk.Init("___PUBLIC_DSN___");
+    }
+
+    protected void Application_Error()
+    {
+        var exception = Server.GetLastError();
+        // Capture the server errors.
+        SentrySdk.CaptureException(exception);
+    }
+
+    protected void Application_End()
+    {
+        // Flushes out events before shutting down.
+        _sentry?.Dispose();
+    }
+}
+```
+
+### Documentation
+
+Once you've verified the package is initialized properly and sent a test event, consider visiting our [complete ASP.NET docs](https://docs.sentry.io/platforms/dotnet/guides/aspnet/).
```

Comment:
We can delete this. 

I don't see an intentional being sent? For other SDKs, the next lines are (pulling from Browser JS here):

I don't think this applies --> We recommend adjusting the value of `tracesSampleRate` in production. Learn more about configuring sampling in our [full documentation](https://docs.sentry.io/platforms/javascript/performance/sampling/).

But I think we still need this --> Then create an intentional error, so you can test that everything is working:

```js
myUndefinedFunction();
```

If you're new to Sentry, use the email alert to access your account and complete a product tour.

If you're an existing user and have disabled alerts, you won't receive this email.

Reply:
> We can delete this.

Isn't it a good chance to tell the users about the docs? Once they've set the basics up, they should be ready to go.

I'll add a verify step.